apache/zeppelin
# ZEPPELIN-3029. Cannot delete an interpreter

### What is this PR for?

Only copy interpreterSetting from interpreterSettingTemplates when zeppelin is started the first time (interpreter.json doesn't exist)


### What type of PR is it?
[Bug Fix]

### Todos

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3029

### How should this be tested?
* Manually verified.  Delete an interpreter, and then restart zeppelin to check.

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
